test/uk/gov/hmrc/alcoholdutyreturns/service/AuditServiceSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutyreturns.service
+
+import org.mockito.ArgumentMatchers.any
+import play.api.libs.json.{JsObject, Json}
+import uk.gov.hmrc.alcoholdutyreturns.base.SpecBase
+import uk.gov.hmrc.alcoholdutyreturns.config.AppConfig
+import uk.gov.hmrc.alcoholdutyreturns.models.audit.AuditReturnStarted
+import uk.gov.hmrc.alcoholdutyreturns.models.audit.AuditType.ReturnStarted
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
+import uk.gov.hmrc.play.audit.AuditExtensions._
+import uk.gov.hmrc.play.audit.http.connector.AuditResult.{Disabled, Failure, Success}
+import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
+
+import java.time.{Instant, LocalDate}
+import scala.concurrent.Future
+
+class AuditServiceSpec extends SpecBase {
+  "AuditService" should {
+    "return true if Success is returned from the connector" in new SetUp {
+      when(auditConnector.sendExtendedEvent(any)(any, any)).thenReturn(Future.successful(Success))
```

Comment:
Would have liked to check the event is being passed, but I don't think mocking and JsObject work with regards to equality.